Update meta-omics special interest group to include diagram of associated pipelines (#2665)
nf-core/website: sites/main-site/src/content/special-interest-groups/meta-omics/index.mdx

1. Helping coordinate compatibility between existing and potential meta-omic nf-core pipelines
2. Implementing semi-'chaining' of sequential pipelines
3. Developing standard parameters for different sub-disciplines (i.e. microbiome analysis) for each pipeline

> 4. Documenting usage of 'first level' meta-omic pipelines (createtaxdb, ampliseq, eager, taxprofiler, mag, metatdenovo) followed by complementary analyses with "secondary" pipelines such as (funcscan, phageannotator, phyloplace, rnaseq, differentialabundance) etc.

Author: ```suggestion
4. Documenting usage of 'first level' meta-omic pipelines (createtaxdb, ampliseq, eager, taxprofiler, mag, metatdenovo) followed by complementary analyses with "secondary" pipelines (e.g., funcscan, phageannotator, phyloplace, rnaseq, differentialabundance) etc.
```

<Image
    src={metro-omics-metro}
    class="d-block m-auto"
    width={400}
    density={[1.5, 2]}

>     alt="London Underground style 'tube map' showing how potentially nf-core pipelines can be used for complementary and sequential analysis within meta-omics disciplines"

Author: ```suggestion
    alt="Diagram showcasing how nf-core pipelines can potentially be used for complementary and sequential analysis within meta-omics disciplines using a London Underground style 'tube map'"
```

/>

> ![](nf-core-meta-omics-metro.png)

Author: ```suggestion
```
